### PR TITLE
Add note about User support for LiteBans

### PR DESCRIPTION
**Pull request by Andre601**
LiteBans does not have the best way of support, requiring authentication of purchase to gain access to their main source of support (Their Discord server). This extends to API support too.

I assume LibertyBans provides support through issues on GitHub? If not can I alter it.

### CODE REVIEW

File: docs/Comparison-to-LiteBans.md [hunk 1]
```diff
@@ -61,6 +61,12 @@ LiteBans is also behind a pay-wall, meaning that users have to pay in order to u
 
 Users of LiteBans must carefully consider whether they are willing to rely entirely on the author of LiteBans for features and improvements.
 
+### Support
+
+Support for LibertyBans is provided through GitHub Issues and a dedicated text channel on a Discord Server.
+
+LiteBans primarely offers support through Discord, however, only does so for users who verified their purchase of the plugin. This also includes requesting support for their API, meaning if you're not a customer of their plugin, chances are low you will gain access to their support server, or receive help with their API at all.
```
Comment (A248) [inline docs/Comparison-to-LiteBans.md:68]:
```suggestion
LiteBans primarily offers support through Discord; however, it only does so for users who verified their purchase of the plugin. This also includes requesting support for their API, meaning if you're not a customer of their plugin, chances are low you will gain access to their support server, or receive help with their API at all.
```

**Review by A248 — APPROVED** [review]
Very good addition, thank you. If you like, you can explain why this is detrimental to developers, and why developers might need API support despite not being official "customers." Some intersectional mention of the LiteBans API's lack of quality could emphasize the importance of offering support to developers.

**Comment by Andre601** [comment]
> Very good addition, thank you. If you like, you can explain why this is detrimental to developers, and why developers might need API support despite not being official "customers." Some intersectional mention of the LiteBans API's lack of quality could emphasize the importance of offering support to developers.

Was busy, so hadn't time to respond. I think that would be a good addition to the info, as it clarifies things a bit more.

Tho, is a simple paragraph good for this? Or would a callout or similar be more useful for this?

**Comment by A248** [comment]
Your call. It is a literary work ;)

**Pull request merged**